core/consensus/slots/src/lib.rs

```diff
@@ -76,6 +78,7 @@ pub fn start_slot_worker<B, C, W, T, SO, SC>(
 	sync_oracle: SO,
 	inherent_data_providers: InherentDataProviders,
 	timestamp_extractor: SC,
+	slots_per_epoch: NonZeroU64,
```

Comment:
I think this is unused because we never look at the epoch from `SlotInfo` (and we shouldn't given @rphmeier's comments above).

Reply:
Removed.